drivers/gpio/gpio_mchp_xec.c

```diff
@@ -41,24 +41,39 @@ static int gpio_xec_configure(struct device *dev,
 			      int access_op, u32_t pin, int flags)
 {
 	const struct gpio_xec_config *config = dev->config->config_info;
+	u32_t port_n = config->port_num;
 	__IO u32_t *current_pcr1;
-	u32_t pcr1 = 0;
-	u32_t mask = 0;
-	u32_t gpio_interrupt = 0;
+	u32_t pcr1 = 0U;
+	u32_t mask = 0U;
+	__IO u32_t *gpio_base = (__IO u32_t *)(GPIO_PAROUT_BASE
```

Comment:
HAL has MCHP_GPIO_PAROUT_ADDR(n) where n is the port number 0 <= port num < NUM_MCHP_GPIO_PORTS

Reply:
`MCHP_GPIO_BASE_ADDR` is not defined so those macros cannot be used.